core/include/kernel/tee_time.h

```diff
@@ -38,5 +38,6 @@ TEE_Result tee_time_get_ta_time(const TEE_UUID *uuid, TEE_Time *time);
 TEE_Result tee_time_get_ree_time(TEE_Time *time);
 TEE_Result tee_time_set_ta_time(const TEE_UUID *uuid, const TEE_Time *time);
 void tee_time_wait(uint32_t milliseconds_delay);
+void tee_time_mdelay(uint32_t ms);
```

Comment:
Please add /* Busy wait */ before the declaration to make it clear how it's different from `tee_time_wait()`.